_private/CreateSession.html

```diff
@@ -0,0 +1,147 @@
+						<div class="col-xs-4 col-xs-offset-2 pull-right">
+							<input onclick="showTeamOptions()" type="checkbox" value="1" name="Teams" />
+							<label for="Teams">Teams</label>
+						</div>
+					</div>
+
+					<!--This gets hidden when the Teams checkbox is clicked. -->
+					<div id="options_max_players" class="form-group row">
+						<label class="col-sm-3 col-form-label" for="max_players">Max Players</label>
+						<div class="col-sm-9">
+							<input class="form-control" type="number" id="max_players" placeholder="25" name="max_players" />
+						</div>
+					</div>
+
+					<!--This gets hidden when the Teams checkbox is unclicked. -->
+					<div id="options_teams" style="display:none">
+						<div class="form-group row">
+							<label class="col-sm-3 col-form-label" for="num_teams">Teams</label>
+							<div class="col-sm-9">
+								<input class="form-control" type="number" id="num_teams" placeholder="3" name="num_teams" />
+							</div>
+						</div>
+						<div class="form-group row">
+							<label class="col-sm-3 col-form-label" for="max_players_teams">Max players per team</label>
+							<div class="col-sm-9">
+								<input class="form-control" type="number" id="max_players_teams" placeholder="3" name="max_players_teams" />
+							</div>
+						</div>
+						<div class="form-group row">
+							<label class="col-sm-3 col-form-label" for="team_leader_selection">Team leader selection</label>
+							<div class="col-sm-9">
+								<select class="form-control" id="team_leader_selection">
+									<option>Random</option>
+									<option>Player Choice</option>
+								</select>
+							</div>
+						</div>
+						<div class="form-group row">
+							<label class="col-sm-3 col-form-label" for="team_name_selection">Team name selection</label>
+							<div class="col-sm-9">
+								<select class="form-control" id="team_name_selection">
+									<option value="Auto">Auto</option>
+									<option value="Manual"">Manual</option>
+								</select>
+							</div>
+						</div>
+						<!--Usually hidden unless Manual is chosen from the team name selection -->
+						<div id="optionsManualNameSelection" style="display:none" >
+
+						</div>
+					</div>
+
+					<!--Submit and cancel buttons -->
+					<div class="form-group row">
+						<div class="col-xs-9 col-xs-offset-4">
+							<input type="submit" value="Create" class="btn btn-primary" />
+							<input type="button" value="Cancel" class="btn" />
+						</div>
+					</div>
+				</form>
+			</div>
+		</div>
+	</div>
+
+	<script src="http://code.jquery.com/jquery-latest.min.js"></script>
+	<script src="bootstrap/js/bootstrap.min.js"></script>
+	<script type="text/javascript">
+		
+		function showTeamOptions()
+		{
+			$('#options_max_players').toggle('slow');
+			$('#options_teams').toggle('slow');
+		}
+
+		$("#num_teams").on('keyup mouseup',function(){
+			$('#optionsManualNameSelection').empty();
+			var loops = $('#num_teams').val();
+
+			for(x = 1; x<= loops;x++)
```

Comment:
Two things. Generally developers expect the counter to be a zero based (unless from unholy languages), so making the counter 1 can lead to accidental logically bugs. I would suggest making it zero based and adding one when rendering for non-programmers (aka simple humans :D). Also `x` why not `i`?